packages/core/admin/server/tests/admin-api-token.test.e2e.js

```diff
@@ -202,4 +204,29 @@ describe('Admin API Token CRUD (e2e)', () => {
     expect(res.statusCode).toBe(200);
     expect(res.body.data).toBeNull();
   });
+
+  test('9. Retrieves a token (successfully)', async () => {
+    const res = await rq({
+      url: '/admin/api-tokens/1',
+      method: 'GET',
+    });
+
+    expect(res.statusCode).toBe(200);
+    expect(res.body.data).toStrictEqual({
+      name: 'api-token_tests-name',
+      description: 'api-token_tests-description',
+      type: 'read-only',
+      id: 1,
```

Comment:
Can you use the id return by the created api-token ? I think it's better as the id depends if the database is fresh

Reply:
I'll update the test suite to be less dependent on hardcoded ids 👌